changed.d

```diff
@@ -1,27 +1,36 @@
 #!/usr/bin/env rdmd
+
 // Written in the D programming language
 
 /**
 Change log generator which fetches the list of bugfixes
 from the D Bugzilla between the given dates.
+Moreover manual changes are accumulated from raw text files in the
+Dlang repositories.
 It stores its result in DDoc form to a text file.
 
-Copyright: Dmitry Olshansky 2013.
+Copyright: D Language Foundation 2016.
```

Comment:
Sure whatever. When I started this script it was a brutal hack to show that automation is possible and easy.